app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity_table = 8
+        self.threshold_table = int(self.capacity_table * (2 / 3))
+        self.size_table = 0
+        self.hash_table = [[] for i in range(self.capacity_table)]
+
+    def __len__(self) -> int:
+        return self.size_table
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.size_table == self.threshold_table:
+            self.resize_table()
+        hash_key = hash(key)
+        self.fill_table(key, value, hash_key)
+
+    def __getitem__(self, key: Hashable) -> list | None:
+        hash_key = hash(key)
+        hash_ind = hash_key % self.capacity_table
```

Comment:
I don't think `ind` is a very good abbreviation. It only saves you two characters anyway. So why don't you rename the variable into `hash_index`? Same at the line 43.